protobufs/types.proto

```
@@ -8,124 +8,142 @@ package proto;
-// ControllerSticks message
-message ProtoControllerStick {
-  string type = 1; // Fixed value "ControllerStick"
-  int32 slot = 2; // Slot number (0-3)
-  int32 stick = 3; // Stick number (0 for left, 1 for right)
-  int32 x = 4; // X axis value (-32768 to 32767)
-  int32 y = 5; // Y axis value (-32768 to 32767)
+/* WebRTC + signaling */
+
+message RTCIceCandidateInit {
+  string candidate = 1;
+  optional uint32 sdpMLineIndex = 2;
```

Buf (1.59.0)

Field name "sdpMLineIndex" should be lower_snake_case, such as "sdp_m_line_index".

(FIELD_LOWER_SNAKE_CASE)